mon-entreprise/source/components/SchemeComparaison.tsx

```diff
@@ -405,60 +407,66 @@ export default function SchemeComparaison({
 								/>
 							</>
 						</div>
-						<h3 className="legend">
-							<Trans i18nKey="comparaisonRégimes.retraiteEstimation.legend">
-								<span>Pension de retraite</span>
-								<small>(avant impôts)</small>
-							</Trans>
-						</h3>
-						<div className="AS">
-							<Value
-								linkToRule={false}
-								engine={assimiléEngine}
-								precision={0}
-								expression="protection sociale . retraite"
-							/>{' '}
-							<InfoBulle>
-								<Trans i18nKey="comparaisonRégimes.retraiteEstimation.infobulles.AS">
-									Pension calculée pour 172 trimestres cotisés au régime général
-									sans variations de revenus.
-								</Trans>
-							</InfoBulle>
-						</div>
-						<div className="indep">
-							<Value
-								linkToRule={false}
-								engine={indépendantEngine}
-								precision={0}
-								expression="protection sociale . retraite"
-							/>{' '}
-							<InfoBulle>
-								<Trans i18nKey="comparaisonRégimes.retraiteEstimation.infobulles.indep">
-									Pension calculée pour 172 trimestres cotisés au régime des
-									indépendants sans variations de revenus.
-								</Trans>
-							</InfoBulle>
-						</div>
-						<div className="auto">
-							{plafondAutoEntrepreneurDépassé ? (
-								'—'
-							) : (
-								<>
+						{indépendantEngine.evaluate(
+							'entreprise . activité . libérale réglementée'
+						).nodeValue !== true && (
+							<>
+								<h3 className="legend">
+									<Trans i18nKey="comparaisonRégimes.retraiteEstimation.legend">
+										<span>Pension de retraite</span>
+										<small>(avant impôts)</small>
+									</Trans>
+								</h3>
+								<div className="AS">
 									<Value
 										linkToRule={false}
-										engine={autoEntrepreneurEngine}
+										engine={assimiléEngine}
 										precision={0}
 										expression="protection sociale . retraite"
 									/>{' '}
 									<InfoBulle>
-										<Trans i18nKey="comparaisonRégimes.retraiteEstimation.infobulles.auto">
-											Pension calculée pour 172 trimestres cotisés en
-											auto-entrepreneur sans variations de revenus.
+										<Trans i18nKey="comparaisonRégimes.retraiteEstimation.infobulles.AS">
+											Pension calculée pour 172 trimestres cotisés au régime
+											général sans variations de revenus.
 										</Trans>
 									</InfoBulle>
-								</>
-							)}
-						</div>
+								</div>
+								<div className="indep">
+									<Value
+										linkToRule={false}
+										engine={indépendantEngine}
+										precision={0}
+										expression="protection sociale . retraite"
+									/>{' '}
+									<InfoBulle>
+										<Trans i18nKey="comparaisonRégimes.retraiteEstimation.infobulles.indep">
+											Pension calculée pour 172 trimestres cotisés au régime des
```

Comment:
```suggestion
											Pension calculée à titre indicatif pour 172 trimestres cotisés au régime des
```

Pour bien insister sur le fait qu'en pratique on a pas assez d'impôt pour faire un calcul précis ?